docs/source/guide.rst

```diff
@@ -379,78 +379,79 @@ Withdraw function needs amount of ether to withdraw as a input argument.
 History
 =======
 
-To access list of user's transactions (with all statuses) use ``useTransactions`` hook.
-Transactions are stored in local storage and the status is rechecked on every new block. 
-
-Take a look at example usage below:
+To access history of transaction sent by useDApp, use :ref:`useTransactions` hook.
 
 .. code-block:: javascript
 
   const { transactions } = useTransactions()
 
+``transactions`` is an array so you can use ``transactions.map(...)`` to display all of 
+transactions.
```

Comment:
I would add links to core docs

Reply:
There is a link to core docs on line 382, or should it be a different link ?